internal/tiltfile/tiltfile_test.go

```diff
@@ -317,7 +317,7 @@ func TestGetManifestConfigNonFunction(t *testing.T) {
 func TestGetManifestConfigTakesArgs(t *testing.T) {
 	file := tempFile(
 		`def blorgly2(x):
-			return "foo"
+      return "foo"
```

Comment:
My editor had some weird interaction with indentation, tabs vs spaces and `gofmt`.